pkgs/by-name/ra/radicle-tui/package.nix

```
@@ -0,0 +1,77 @@
+    hash = "sha256-rz9l9GtycqZoROUI6Hn0Fv5Br0YCIrcHlEWLMP4hasQ=";
+    leaveDotGit = true;
+    postFetch = ''
+      git -C $out rev-parse HEAD > $out/.git_head
+      git -C $out log -1 --pretty=%ct HEAD > $out/.git_time
+      rm -rf $out/.git
+    '';
+  };
+
+  cargoHash = "sha256-f9D4RKWw7y6z9rERuF7F6soyNITvKa6QVt34biZZ5JY=";
+
+  nativeCheckInputs = [
+    radicle-node
+  ];
+
+  checkFlags = [
+    "--skip=cli::can_be_executed"
+    "--skip=cli::empty_command_is_forwarded"
+    "--skip=cli::help_command_is_forwarded"
+    "--skip=cli::version_command_is_forwarded"
+    "--skip=commands::tui_inbox::cli::empty_operation"
+    "--skip=commands::tui_inbox::cli::empty_operation_is_not_forwarded"
+    "--skip=commands::tui_inbox::cli::empty_operation_is_not_forwarded_explicitly"
+    "--skip=commands::tui_inbox::cli::empty_operation_with_help_is_forwarded"
+    "--skip=commands::tui_inbox::cli::list_operation_is_not_forwarded"
+    "--skip=commands::tui_inbox::cli::list_operation_is_not_forwarded_explicitly"
+    "--skip=commands::tui_inbox::cli::list_operation_with_help_is_forwarded"
+    "--skip=commands::tui_inbox::cli::unknown_operation_show_is_forwarded"
+    "--skip=commands::tui_patch::cli::empty_operation"
+    "--skip=commands::tui_patch::cli::empty_operation_is_not_forwarded"
+    "--skip=commands::tui_patch::cli::empty_operation_is_not_forwarded_explicitly"
+    "--skip=commands::tui_patch::cli::empty_operation_with_help_is_forwarded"
+    "--skip=commands::tui_patch::cli::list_operation_is_not_forwarded"
+    "--skip=commands::tui_patch::cli::list_operation_is_not_forwarded_explicitly"
+    "--skip=commands::tui_patch::cli::list_operation_with_help_is_forwarded"
+    "--skip=commands::tui_patch::cli::unknown_operation_edit_is_forwarded"
+    "--skip=commands::tui_patch::cli::unknown_operation_is_not_forwarded"
+    "--skip=commands::tui_patch::cli::unknown_operation_show_is_forwarded"
+    "--skip=commands::tui_patch::review::test::app_with_single_file_multiple_hunks_can_be_constructed"
+    "--skip=commands::tui_patch::review::test::app_with_single_hunk_can_be_constructed"
+    "--skip=commands::tui_patch::review::test::first_hunk_is_selected_by_default"
+    "--skip=commands::tui_patch::review::test::hunk_can_be_selected"
+    "--skip=commands::tui_patch::review::test::hunks_are_rejected_by_default"
+    "--skip=commands::tui_patch::review::test::multiple_files_single_hunk_can_be_accepted"
+    "--skip=commands::tui_patch::review::test::single_file_multiple_hunks_only_last_can_be_accepted"
+    "--skip=commands::tui_patch::review::test::single_file_single_hunk_can_be_accepted"
+  ];
+
+  buildInputs = lib.optionals stdenv.isDarwin [ libiconv ];
+
+  meta = {
+    description = "Radicle TUI";
+    homepage = "https://radicle.xyz";
+    license = with lib.licenses; [
+      asl20 # or
+      mit
+    ];
+    maintainers = with lib.maintainers; [ matthiasbeyer ];
+    mainProgram = "radicle-tui";
```

```suggestion
    mainProgram = "rad-tui";
```

Specified in `Cargo.toml`:

```toml
[[bin]]
name = "rad-tui"
path = "bin/main.rs"
required-features = ["bin"]
```